packages/core/src/index.ts

```diff
@@ -675,6 +676,10 @@ class Hopr extends EventEmitter {
     return ethereum.withdraw(currency, recipient, amount)
   }
 
+  public emitMessage(msg: Uint8Array) {
```

Comment:
why is this here, and public?

Reply:
... because it is I need it in the forward logic, but could be solved differently. Do not hesitate to propose sth else

Reply:
why not passed in with a private function as before?

Reply:
Please be more specific and post links to the code

Reply:
solved, method was not used, now obsolete